deploy/gcp/README.md

```diff
@@ -43,11 +43,6 @@ This guide assumes you already have a functional Kubernetes cluster
 and helm installed (see below for instructions on installing
 helm). Note the following requirements:
 
-* The helm chart deploys Prometheus and Grafana to collect and display
```

Comment:
Keep this paragraph

Reply:
Why do you say that? There is a section below talking about it. Perhaps you mean that it should say "To use this helm chart you must install Prpmetheus and Grafana in your cluster as described below. Your cluster must contain..."

But you can't just delete the entire bullet point. You also need to add a similar bullet point in the AWS readme with a machine type that is appropriate for AWS.

Reply:
@arunraman  Can you just revert this file to how it was before your changes. You've replaced the existing bullet with new text that has less information. We specifically have the machine type in the description because of customer feedback.